### PR TITLE
I have added hero and about me sections and updated the previous source code.

### PR DESCRIPTION
**Pull request by rozhi-98**
When I started the project, I did not consider responsiveness of the website. Everything was hard coded with pixels including width and height of the page. I realized it is not a good idea since we will be working on this project for a long time. Hence, I have updated the whole source code even for the header section which was included in the first version of the project.

### CODE REVIEW

**Comment by Gopxfs** [comment]
# STATUS: INVALID REQUEST

Hi @rozhi-98 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project without the linters passing. Please check your linters and fix the errors 😄.

Your Code Review Request will be marked invalid in your Dashboard, so please submit a new one once ready 🙏.
Invalid requests don't count as a review, so don't worry about that 😄!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -4,72 +4,396 @@
...
+            <p class="card-info">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="card-tags">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <a href="#" class="button">See Project</a>
```
Comment (Olamarx) [inline index.html:163]: - [ ] Kindly add the styling to fit in as in the image below. Kindly apply it to all buttons in your app.
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @rozhi-98, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good commit messages ✔️ 
- Good use of Git and GitHub flows ✔️ 
- All linter passes 👍🏼 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @rozhi-98, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations


## Good Highlights✔

- Linter checks are passing
- Adherence to GitHub flow
- Good commit messages
- Request change has been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**